packages/cursorless-engine/src/processTargets/targets/BaseTarget.ts

```diff
@@ -39,7 +51,7 @@ export default abstract class BaseTarget implements Target {
   isNotebookCell = false;
   isWord = false;
 
-  constructor(parameters: CommonTargetParameters) {
+  constructor(parameters: TParameters & CommonTargetParameters) {
```

Comment:
This ensures that whatever type is declared actually ends up existing in the subclass constructor somehow.

Reply:
oh really?  so the type signature of the base class constructor influences the type signature of derived class constructors?  That's surprising to me